### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by ddddqt** [joslefaure/HIT#43]
Hello author, I am very interested in your work and see that your code uses a loss function of your own design, but the article does not mention what loss function you use in your work. Can you tell me what method you use for this loss function.Thank you very much!

**Comment by joslefaure** [comment]
The loss function is not of my own design. It's a softmax focal loss

**Comment by ddddqt** [comment]
> The loss function is not of my own design. It's a softmax focal loss

Okay, I see that. Thank you very much
